### DISCUSSION TITLE
Taskbar features and settings

### DISCUSSION THREAD

**Discussion by CorruptionVR** [DustinBrett/daedalOS#281] [Feature Requests]
here are some features that I would like added to the taskbar. I like that the webOS is based on windows 10, and it looks really good.

- [ ] move the taskbar to one side of the screen(bottom, top, left, right)
- [x]  search button next to start button, would open query in browser
- [ ] search bar
- [ ] task view
- [ ] resize taskbar, either by dragging the top like in windows 10, or by inputting height in pixels in prompt
- [ ] show all applications
- [ ] tiles in start menu(low priority)
- [ ] icon only taskbar(no labels)
- [ ] pin applications
- [ ] change start menu icon

**Comment by DustinBrett** [comment]
Thanks for these suggestions. I will keep these in mind and most of these I have indeed wanted to do and hope to one day.
